src/room/InCallView.module.css

```diff
@@ -144,3 +144,28 @@ Please see LICENSE in the repository root for full details.
   position: relative;
   flex-grow: 1;
 }
+
+.floatingReaction {
+  position: relative;
+  display: inline;
+  z-index: 2;
+  font-size: 32pt;
+  /* Reactions are "active" for 3 seconds (as per REACTION_ACTIVE_TIME_MS), give a bit more time for it to fade out. */
+  animation-duration: 4s;
+  animation-name: reaction-up;
+  width: fit-content;
+}
+
+@keyframes reaction-up {
+  from {
+    opacity: 1;
+    translate: 100vw 0;
+    scale: 200%;
+  }
+
+  to {
+    opacity: 0;
+    translate: 100vw -100vh;
+    scale: 100%;
+  }
+}
```

Comment:
Accessibility: Ideally this would skip the `translate` part of the animation (and maybe `scale` too) if `@media (prefers-reduced-motion)` matches

Reply:
Excellent idea!